src/documents/ChangeApplierAPI.md

```diff
@@ -8,9 +8,9 @@ This section explains and documents the various Javascript API calls for instant
 
```

Comment:
In the opening paragraph, it would be good if the first use of "ChangeApplier" was a link to the ChangeApplier page.